### DISCUSSION TITLE
Where do I find Vercel REST API's openapi spec?

### DISCUSSION THREAD

**Discussion by tonyxiao** [vercel/vercel#11682] [Help]
### Question

The documentation is clearly generated from spec, but the spec itself does not appear to be accessible. Did I miss  it somewhere? Or can the team expose the spec please?

**Comment by tonyxiao** [comment]
Found it at https://openapi.vercel.sh/, which I assume is legit? This was via apis.guru. I'm surprised this isn't documented somewhere...

**Reply by leerob** [reply]
Can you share more about what you're looking for? The API docs are auto-generated from the spec.

**Reply by tonyxiao** [reply]
I'm looking to use my own SDK generator (https://github.com/tonyxiao/openSDKs) to interact with the Vercel API. I work with dozens of APIs and do not wish to use bespoke SDK for each API.
